frontend/pages/project-developer/[id]/index.tsx

```diff
@@ -1,108 +1,126 @@
+import { CategoryType } from 'types/category';
+import { GroupedEnums as GroupedEnumsType } from 'types/enums';
+import { ProjectDeveloper as ProjectDeveloperType } from 'types/projectDeveloper';
+
+import { getEnums } from 'services/enums/enumService';
+import { getProjectDeveloper } from 'services/project-developers/projectDevelopersService';
+
+export const getServerSideProps = async ({ params: { id }, locale }) => {
+  let projectDeveloper;
+
+  // If getting the project developer fails, it's most likely because the record has
+  // not been found. Let's return a 404. Anything else will trigger a 500 by default.
+  try {
+    ({ data: projectDeveloper } = await getProjectDeveloper(id, { includes: 'projects' }));
+  } catch (e) {
+    return { notFound: true };
+  }
+
+  const enums = await getEnums();
 
-export async function getStaticProps(ctx) {
   return {
     props: {
-      intlMessages: await loadI18nMessages(ctx),
+      intlMessages: await loadI18nMessages({ locale }),
+      enums: groupBy(enums, 'type'),
+      // Fixing issues with circular references (caused by the inclusion of projects)
+      // https://github.com/vercel/next.js/discussions/10992#discussioncomment-59574
+      projectDeveloper: decycle(projectDeveloper),
     },
   };
-}
+};
 
-export async function getStaticPaths() {
-  return {
-    paths: [],
-    fallback: true,
-  };
-}
-
-const InvestorPage: PageComponent<{}, StaticPageLayoutProps> = (props) => {
-  const intl = useIntl();
-  // const { query } = useRouter();
-  // const { id } = query;
-
-  const aboutInfo: {
-    logo: string;
-    name: string;
-    description: string;
-    text: string;
-    website: string;
-    social: SocialType[];
-    contact: string;
-  } = {
-    name: 'Herencia Columbia',
-    description: 'Non Governamental Agency',
-    text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Iaculis gravida auctor enim, id nisl nisl sem tristique. Rhoncus vestibulum vitae diam dignissim imperdiet. Lacus, morbi non cras maecenas cras scelerisque eget. Rutrum tincidunt sed elit rhoncus nunc nisl pulvinar consectetur tincidunt. Nunc quisque potenti velit suscipit volutpat tellus',
-    logo: '/images/placeholders/profile-logo.png',
-    website: 'https://www.site.com',
-    social: [
-      { id: 'linked-in', url: 'https://www.linkedin.com' },
-      { id: 'twitter', url: 'https://www.twitter.com' },
-      { id: 'facebook', url: 'https://www.facebook.com' },
-      { id: 'instagram', url: 'https://www.instagram.com' },
-    ],
-    contact: 'Joel Bean',
+type ProjectDeveloperPageProps = {
+  projectDeveloper: ProjectDeveloperType;
+  enums: GroupedEnumsType;
+};
+
+const ProjectDeveloperPage: PageComponent<ProjectDeveloperPageProps, StaticPageLayoutProps> = ({
+  projectDeveloper,
+  enums,
+}) => {
+  const projectDeveloperTypeName = enums[EnumTypes.ProjectDeveloperType].find(
+    ({ id }) => id === projectDeveloper.project_developer_type
+  )?.name;
+
+  const funding = {
```

Comment:
Is it worth it to use `useMemo` for these? 

Considering the content is virtually static, it may not be worth the extra computations; the page should not re-render.